README.md

```diff
@@ -1,2 +1,125 @@
-# extension-boilerplate
-extension boiler plate written in typescript and svelte
+<br /><br />
+<p align="center">
+<a href="https://osvauld.com">
+  <img src="https://raw.githubusercontent.com/tonyantony300/crispy-guacamole/master/assets/logo.png" alt="Osvauld Logo" width="70">
+</a>
+</p>
+
+<p align="center"><b>Credential management for teams in the most secure way</b></p>
+
+
+<p>
+   <a href="https://app.osvauld.com/#gh-light-mode-only" target="_blank">
+      <img
+        src="https://raw.githubusercontent.com/tonyantony300/crispy-guacamole/master/assets/Credential_Single_Card.png"
+        alt="Osvauld Screens"
+        width="100%"
+      />
+    </a>
+        <a href="https://osvauld.com" target="_blank">
+      <img
+        src="https://raw.githubusercontent.com/tonyantony300/crispy-guacamole/master/assets/Credential_Single_Card.png"
+        alt="Osvauld Issue Details"
+        width="100%"
+      />
+    </a>
+
+</p>
+
+
+Meet [Osvauld](https://osvauld.com). An open-source credential management tool intended to solve password-mess teams face. Osvauld helps to enforce password/token hygiene and visibilty across your team thus resulting in better overall security.
+
+> Osvauld is still in its early days, not everything will be perfect yet, and hiccups may happen. Please let us know of any suggestions, ideas, or bugs that you encounter on our [Discord](https://discord.gg/XU22FscZ) or GitHub issues, and we will use your feedback to improve on our upcoming releases.
+
+The easiest way to get started with Osvauld is by creating a [Osvauld Cloud](https://app.osvauld.com) account. Osvauld Cloud offers a hosted solution for Osvauld. If you prefer to self-host Osvauld, please refer to our [deployment documentation](https://docs.Osvauld.so/docker-compose). 
```

Comment:
we dont have cloud yet.

Reply:
Yeah, we will tell its a place holder if someone asks, which is unlikely at this point